src/style.css

```diff
@@ -0,0 +1,129 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+body {
+  margin: 0 auto;
+}
+
+.big__form {
+  position: relative;
+  box-sizing: border-box;
+  width: calc(100% - 50px);
+  height: 70px;
+  left: 50px;
+  top: 20px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background: #fff;
+}
+
+.big__form::before {
+  display: block;
+  position: absolute;
+  content: "";
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+  background: url(/images/Search.svg);
+}
```

Comment:
I suggest you make input the size of the whole form, this way it'll be positioned correctly, and for magnifying glass use absolutely positioned button